app/Http/Controllers/Store/DashboardController.php

```diff
@@ -25,6 +25,7 @@ public function index()
             URL::route('store.registrations.print');
 
         $data = [
+            "user" => $user,
```

Comment:
If you are passing whole `$user`, do you still need `$user->name`?

Reply:
entirely for convenience and clarity in the blade; there's an include to make the top-right box with the current user name that requires it, and is included everywhere. I didn't want to have to check a whole bunch of blades at this point